src/main/java/me/botsko/prism/commands/AboutCommand.java

```diff
@@ -28,12 +28,12 @@ public AboutCommand(Prism plugin) {
 	@Override
 	public void handle(CallInfo call) {
 		call.getSender().sendMessage(Prism.messenger.playerHeaderMsg(
-				"Prism - By " + ChatColor.GOLD + "viveleroi" + ChatColor.GRAY + " v" + plugin.getPrismVersion()));
```

Comment:
Is there any possibility to retain something that says, "originally by viveleroi" or something? Maybe on the next line?

Reply:
Yep, we can do that :slightly_smiling_face: added in #146

Reply:
I only really wanted to remove it so the people looking for support knew where to look